omnigibson/scenes/scene_base.py

```diff
@@ -185,8 +185,10 @@ def _load(self):
             )
             og.sim.import_object(self._skybox, register=False)
             light_prim = self._skybox.light_link.prim
-            light_prim.GetAttribute("color").Set(Gf.Vec3f(1.07, 0.85, 0.61))
-            light_prim.GetAttribute("texture:file").Set(Sdf.AssetPath(m.DEFAULT_SKYBOX_TEXTURE))
+            color_key = "color" if "color" in light_prim.GetPropertyNames() else "inputs:color"
+            light_prim.GetAttribute(color_key).Set(Gf.Vec3f(1.07, 0.85, 0.61))
+            texture_key = "texture:file" if "texture:file" in light_prim.GetPropertyNames() else "inputs:texture:file"
+            light_prim.GetAttribute(texture_key).Set(Sdf.AssetPath(m.DEFAULT_SKYBOX_TEXTURE))
```

Comment:
nit, I wonder if there's a more programmatic way to do this, e.g. something like:

```python
if version >= 2023.1.0:
    append "input" to front of attribute
```

Reply:
Done.